azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellViewHolder.kt

```diff
@@ -73,10 +73,22 @@ internal class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(it
                 )
             )
         }
-        if (accessoryImage.contentDescription != null) {
-            accessoryImage.contentDescription = bottomCellItem.accessoryImageDescription
-        }
-        accessoryImage.visibility = if (bottomCellItem.enabled == true) View.VISIBLE else View.INVISIBLE
+        accessoryImage.contentDescription = bottomCellItem.accessoryImageDescription
+        accessoryImage.visibility =
+            if (isAccessoryImageViewable(bottomCellItem)) View.VISIBLE else View.INVISIBLE
         onClickAction = bottomCellItem.onClickAction
     }
+
+    private fun isAccessoryImageViewable(bottomCellItem: BottomCellItem): Boolean {
+        val muteDescription = itemView.rootView.context
+            .getString(R.string.azure_communication_ui_calling_view_participant_list_muted_accessibility_label)
+        val unmutedDescription = itemView.rootView.context
+            .getString(R.string.azure_communication_ui_calling_view_participant_list_unmuted_accessibility_label)
+
+        if (bottomCellItem.enabled == true) return true
```

Comment:
Use {} for this branching if/else/elseif

Reply:
also, just need one if and ||, not a else if.

Reply:
can actually just return the boolean.

e.g. return   (bottomCellItem.enabled || bottomCellItem.accessoryImageDescription == muteDescription || bottomCellItem.accessoryImageDescription == unmutedDescription)